terraform/modules/upload-service/sqs.tf

```diff
@@ -0,0 +1,58 @@
+resource "aws_sqs_queue" "upload_queue" {
+  name                      = "pre_checksum_upload_queue"
+//  Queue visibility timeout must be larger than (triggered lambda) function timeout
+  visibility_timeout_seconds = 360
+  message_retention_seconds = 86400
+  redrive_policy            = "{\"deadLetterTargetArn\":\"${aws_sqs_queue.deadletter_queue.arn}\",\"maxReceiveCount\":4}"
+
+}
+
+
+resource "aws_sqs_queue" "deadletter_queue" {
+  name                      = "pre_checksum_upload_deadletter_queue"
```

Comment:
as mentioned at another place, please add mention of environment and in general component-functionname-environment is a good practice to follow. So for example upload-pre-checksum-deadletter-queue-predev

Reply:
👍